### DISCUSSION TITLE
Counting State From Organization work

### DISCUSSION THREAD

**Discussion by BurhanUlka** [anuraghazra/github-readme-stats#1278] [Q&A] [answered]
How to add state count which commits  made under any organization............. :) :)

**Comment by rickstaa** [comment, marked as answer]
@BurhanUlka Thanks a lot for your question. This is a highly requested feature see #1 which will likely be merged in the future. You can already use it on your own vercel instance. See https://github.com/anuraghazra/github-readme-stats/issues/1#issuecomment-981419174 for more information.

**Reply by rickstaa** [reply]
@BurhanUlka Did that answer your question?
